contents/handbook/people/team-structure/platform.md

```diff
@@ -44,6 +44,8 @@ Concrete things we take responsibility over:
 
 ## Output metrics
 
+- Primary: Ease of deployment (?)
```

Comment:
I know this isn't feasible for a lot of envs - but pushing for time to deploy could be a good one (lower is better).

Reply:
I don't think time to deploy is a great metric in isolation.
A few candidate metrics could be:
- Number of attempted installs / successful installs (conversion rate)
- NPS for users who have attempted an install (get their email at the start and follow up after a day or so)
- Unique count of customers requiring support getting setup

Reply:
> Number of attempted installs / successful installs (conversion rate)

To me this is best suggestion - NPS / unique count are quite abstracted (and we'll lose lots of data).

Can we implement this funnel - what's the first event we'd capture... Maybe just a visit to the deployment page?

Reply:
I wouldn't go that high level since that's starting to get into the marketing copy.

1) I would say the first intent we would capture is the intent to install - captured when they run the CLI with an install command, that we are planning on building.
2) The next step we would be interested in is that our stack successfully started (instance status reports). 
3) Finally I would look for some retention metric for how long an install remains healthy and in service.

Reply:
Updated in line with this, and have created an [issue to track implementing](https://github.com/PostHog/posthog/issues/7826) this tracking